src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -44,7 +43,6 @@ export class AddBulkConcepts extends Component {
   }
 
   componentDidMount() {
-    this.props.fetchConceptSources();
     this.props.fetchSourceConcepts(this.sourceUrl);
```

Comment:
Good work on this @blackdevelopa.
Is this being used anywhere on the page?

Reply:
Thanks for pointing this out. I removed it completely